Sources/Transaction/TransactionBuilder.swift

```diff
@@ -294,7 +317,13 @@ final class TransactionBuilder {
             fogResolver.withUnsafeOpaquePointer { fogResolverPtr in
                 // Safety: mc_transaction_builder_create should never return nil.
                 withMcInfallible {
-                    mc_transaction_builder_create(fee, tombstoneBlockIndex, fogResolverPtr, memoBuilderPtr, blockVersion)
+                    mc_transaction_builder_create(
+                            fee.value,
+                            tokenId.value,
```

Comment:
Seems strange to discard the tokenId of the Amount parameter (`fee`), but have a separate tokenId param for the builder.

It also would seem strange to use the tokenId of the fee to infer the tokenId for the transaction builder.

Maybe this function should also only take a value param instead of an Amount, so that we're not receiving information that is not used...

Or, we can leave it like it is, and check that the tokenId of Amount matches the tokenId param, but we'd have to throw an exception or handle that error condition somehow.

Reply:
changed the init to use the tokenId from the fee Amount

v1.2.0 does not support mixed transaction so all txOuts and the fee will have the same tokenId